packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -161,17 +160,28 @@ class ChromeLauncher {
       }));
     }
 
+    process.stdout
+      .transform(utf8.decoder)
+      .transform(const LineSplitter())
+      .listen((String line) {
+        globals.printTrace('[CHROME]: $line');
+      });
+
     // Wait until the DevTools are listening before trying to connect.
     await process.stderr
-        .transform(utf8.decoder)
-        .transform(const LineSplitter())
-        .firstWhere((String line) => line.startsWith('DevTools listening'), orElse: () {
-          return 'Failed to spawn stderr';
-        })
-        .timeout(const Duration(seconds: 60), onTimeout: () {
-          throwToolExit('Unable to connect to Chrome DevTools.');
-          return null;
-        });
+      .transform(utf8.decoder)
+      .transform(const LineSplitter())
+      .map((String line) {
+        globals.printTrace('[CHROME]:$line');
+        return line;
+      })
+      .firstWhere((String line) => line.startsWith('DevTools listening'), orElse: () {
+        return 'Failed to spawn stderr';
+      })
+      .timeout(const Duration(seconds: 60), onTimeout: () {
```

Comment:
Aren't we avoiding timeouts in our system?

Reply:
why not

Reply:
I guess if it takes between 50 and 70 seconds it could make tests flaky?